CocoaMarkdown/CMAttributedStringRenderer.m

```diff
@@ -201,7 +207,7 @@ - (void)parser:(CMParser *)parser foundInlineCode:(NSString *)code
 
 - (void)parserFoundSoftBreak:(CMParser *)parser
 {
-    [self appendString:@"\n"];
+    [self appendString:@" "];
```

Comment:
Note that I pulled this from b3b8e36237ed3eef351673b628ee41329eb89961 which appears to be the correct thing to do.